src/pathfinder.js

```diff
@@ -81,8 +82,21 @@ class Pathfinder {
     return cheapestPath
   }
 
+  // params -
+  //   sourceLedger
+  //   destinationLedger
+  //   destinationAmount
+  //   destinationAccount (optional)
```

Comment:
Agree with @clark800, minimize overloading please. Overloading in JS is notoriously brittle. At this stage, I'd rather make a breaking change and always require an object. Another acceptable option is to create a separate method that takes an object.

In general, methods that take a single object as a parameter are well-supported in ES6 thanks to parameter destructuring, so I think it's a good pattern to adopt.

@sentientwaffle: It's good to get into a habit of avoiding breaking changes - so I do like your intent here. Another thought: You could keep the code you have with overloading and just add a deprecation warning to the multiple parameter variant, with the intent to remove it in a couple of weeks.